tower-http/src/timeout/body.rs

```diff
@@ -0,0 +1,379 @@
+//! Middleware that applies a timeout to bodies.
+//!
+//! Bodies must produce data at most within the specified timeout.
+//! If they are inactive, an error will be generated.
+//!
+//! # Differences from `tower_http::timeout::service::Timeout`
```

Comment:
We should use inline links
```suggestion
//! # Differences from [`tower_http::timeout::service::Timeout`]
```